src/utils/css_styler.tsx

```diff
@@ -1,11 +1,36 @@
-// Utility functions to add CSS to HTML style element
+// Utility functions to concatenate view components CSS
+import {IDictionary} from '../models';
 
-let pageCSS = '';
+// Concatenate each component CSS into this variable
+let componentsCSS = '';
 
-export function addCSSToHead(css: string): void {
-  pageCSS += css;
+// Media queries width in px configuration
+const MEDIA_QUERIES_WIDTH_CONFIG: IDictionary<string> = {
+  xsMin : '320px', // Custom, iPhone Retina
+  smMin : '480px', // Extra Small Devices, Phones
+  mdMin : '768px', // Small Devices, Tablets
+  lgMin : '992px', // Medium Devices, Desktops
+  xlgMin: '1200px' // Large Devices, Wide Screens
 }
 
-export function getAllPageHeadCSS(): string {
-  return pageCSS;
+// Supported media queries options
+interface IMediaQueries {
```

Comment:
@phidias 
I'm not sure why, but the compiler won't let me do:
``` ts
// Supported media queries options
interface IMediaQueries {
  xsMin  : string
  smMin? : string
  mdMin? : string
  lgMin? : string
  xlgMin?: string
}
```

If I do it, it throws this error: ``error TS7017: Index signature of object type implicitly has an 'any' type``. Do you know why?